<?xml version="1.0" encoding="UTF-8"?><magicdraw><comment classType="Comment" humanType="Comment" icon="index_files/icon_comment_841774062.jpg" id="_10_5_622020d_1132928308590_561100_5380" refElementId="_10_5_622020d_1132928308590_561100_5380"><annotatedElement humanName="Annotated Element" mode="s,e"><class classType="Class" humanType="Class" icon="index_files/icon_class_555864569.jpg" name="AlreadyConnectedException" refElementId="_9_5_f720368_1110473546643_338677_7869" refid="_9_5_f720368_1110473546643_338677_7869"/></annotatedElement><body humanName="Body" mode="s,e">Unchecked exception thrown when an attempt is made to connect a {@link
SocketChannel} that is already connected.

@version 1.9, 01/11/19
@since 1.4
</body><documentation humanName="Documentation"/><owner classType="Class" humanName="Owner" humanType="Class" icon="index_files/icon_class_555864569.jpg" mode="s,e" name="AlreadyConnectedException" refElementId="_9_5_f720368_1110473546643_338677_7869" refid="_9_5_f720368_1110473546643_338677_7869"/><owningElement classType="Class" humanName="Owning Element" humanType="Class" icon="index_files/icon_class_555864569.jpg" name="AlreadyConnectedException" refElementId="_9_5_f720368_1110473546643_338677_7869" refid="_9_5_f720368_1110473546643_338677_7869"/><TO_DO humanName="To Do" mode="s,e"/></comment></magicdraw>